guides/standalone-inference-scheduling/helmfile.yaml.gotmpl

```diff
@@ -0,0 +1,110 @@
+environments:
```

Comment:
ditto, I don't expect to see anything related to istio or kgateway here

Reply:
done